src/views/publish/measure-revisit.ejs

```diff
@@ -0,0 +1,45 @@
+<%- include("../partials/header", t); %>
+
+<div class="govuk-width-container app-width-container">
+    <!-- <a href="#" class="govuk-back-link">Back</a> -->
```

Comment:
We can remove this I think

Reply:
Removed from here and the dimension revisit page.